specs/applicationdata/ApplicationData.md

```diff
@@ -0,0 +1,428 @@
+      [RoamingSettings](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.roamingsettings),
+      [RoamingStorageQuota](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.roamingstoragequota),
+      [Version](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.version),
+      [SetVersionAsync()](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.setversionasync),
+      [SignalDataChanged()](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.signaldatachanged)
+      and
+      [DataChanged](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.datachanged). See [5.1. Platform Support](#51-platform-support) for more details.
+* **GetDefault() works for all packaged apps** -- Equivalent to .Current but works for all packaged
+  apps (not just callers running in an AppContainer). See
+  [3.3. Reliable access](#33-reliable-access).
+* **`GetForPackageFamily()`** -- Equivalent to Windows.Storage.ApplicationDataManager.CreateForPackageFamily().
+* **`GetForUser()`** -- Synchronous equivalent to GetForUserAsync().
+* **`GetForUnpackaged()`** -- Equivalent functionality for unpackaged apps. See
+  [3.5. Unpackaged app data stores](#35-unpackaged-app-data-stores).
+
+## 3.2. *Path properties
+
+New *Path equivalents to Windows.Storage.ApplicationData's *Folder but returning a `String` rather than `StorageFolder`.
+
+## 3.3. Reliable access
+
+`Windows.Storage.ApplicationData.Current` only works for packaged apps **running in an AppContainer**. Processes not running in an AppContainer (e.g. MediumIL) must use the equivalent `Windows.Management.Core.ApplicationDataManager.CreateForPackageFamily(Windows.ApplicationModel.Package.Current.Id.FamilyName)`.
+
+Microsoft.Windows.Storage.ApplicationData provides consistent behavior as developers expect e.g. `ApplicationData.Current` works regardless if the caller's running in an AppContainer or not.
+
+## 3.4. Per-Machine data store
+
+Package families can optionally have a per-machine data store.
+
+### 3.4.1. Machine Path/Folder
+
+The path for a package family’s "Machine Folder" is located at `%ProgramData%\Microsoft\Windows\AppRepository\ApplicationData\...packagefamilyname...\Machine`.
+
+This directory is ACL’d similarly to a package’s System Metadata directory (`%ProgramData%\Microsoft\Windows\AppRepository\Packages\...pkgfullname...`) e.g.
+
+```
+BUILTIN\Users:(OI)(CI)(Rc,S,RD,REA,X,RA)
+S-1-15-3-2977037414-864741429-1129033548-1928484290-1803339615-1058153653-556172075:(OI)(CI)(RX)
+BUILTIN\Users:(OI)(CI)(R)
+NT SERVICE\TrustedInstaller:(I)(F)
+NT SERVICE\TrustedInstaller:(I)(CI)(IO)(F)
+NT AUTHORITY\SYSTEM:(I)(F)
+NT AUTHORITY\SYSTEM:(I)(OI)(CI)(IO)(F)
+BUILTIN\Administrators:(I)(RX)
+BUILTIN\Administrators:(I)(OI)(CI)(IO)(GR,GE)
+S-1-15-3-1024-3635283841-2530182609-996808640-1887759898-3848208603-3313616867-983405619-2501854204:(I)(RX)
```

Comment:
What are these S-1-15 SIDs? Maybe add a `//` about them?

Reply:
S-1-15-3 is the 'package capability sid'. @oldnewthing has a post about it (of course :P) https://devblogs.microsoft.com/oldnewthing/20220503-00/?p=106557

Though he's slightly mistaken: S-1-15-3 is a package (not app) capability SID. S-1-15-2-... is a PackageSID derived from a package family name, S-1-15-3-1024-... is a 'group' equivalent. The pkgdir is ACL'd with an ACE for the package's S-1-15-3-... ACE.

Reply:
RECOMMEND: Add comment per Raymond's feedback